src/main.js

```diff
@@ -1,11 +1,13 @@
 // The Vue build version to load with the `import` command
 // (runtime-only or standalone) has been set in webpack-base.conf with an alias.
-
 import Vue from 'vue';
 import App from './App';
 import store from './store';
 import GoogleAnalytics from './analytics/GoogleAnalytics';
 
+if ('scrollRestoration' in window.history) {
```

Comment:
Because we use SSR, we can't use client APIs (such as `window`) here because it would break on the server, which runs on Node, and `window` doesn't work there

Reply:
Removed !